oaknational/Oak-Web-Application
# Feat/pupil 944/copyright on quiz results

## Description

Add image attribution information to quiz results view so that it's displayed whenever images appear


## How to test

1. Go to https://deploy-preview-2791--oak-web-application.netlify.thenational.academy/pupils/lessons/the-effects-of-exercise-on-breathing-and-heart-rate
2. Take the starter quiz and observe the image attributions
3. Finish the rest of the lesson to access the review page
4. Observe the image attributions at the bottom of the starter-quiz

## Screenshots

How it used to look (delete if n/a):
<img width="1166" alt="image" src="https://github.com/user-attachments/assets/280baec8-e6ad-4897-8b0f-db41ebcc98ba">

How it should now look:
<img width="1150" alt="image" src="https://github.com/user-attachments/assets/57600e5c-1549-4c3f-b432-cf1e166c3046">



### Acceptance criteria

- [x]  renders copyright text at the bottom of quiz results for quiz questions with images and copyright info
- [x]  renders legal/license text at the bottom of all quiz results
